packages/alfa-rules/src/sia-r10/rule.ts

```diff
@@ -65,7 +60,7 @@ export default Rule.Atomic.of<Page, Attribute>({
       expectations(target) {
         return {
           1: expectation(
-            isValidAutocomplete(target),
+            Attribute.Autocomplete.isValid(target.value),
```

Comment:
We talked about introducing a `hasValidAutocomplete` augmentation function on `Element<"input" | "select" | ...>`, but in the end it seemed a bit out of place to me to use it here; we would have to get `target`'s owner and then cast it (or use `selective`) to get an `Element<"input" | ...>`. 
We could introduce a `hasValidValue` augmentation function on `Attribute` instead, but we would still have to use cast / `selective`...

Reply:
Good point. The cast would be safe due to the filter in the Applicability, but it feels un-needed when looking at the result 👍